schemas/logs/aws/tests/cloud_trail_insight_tests.yml

```diff
@@ -0,0 +1,123 @@
+              "recipientAccountId": "123456789012",
+              "sharedEventID": "12edc982-3348-4794-83d3-a3db26525049",
+              "insightDetails": {
+                  "state": "End",
+                  "eventSource": "ssm.amazonaws.com",
+                  "eventName": "UpdateInstanceAssociationStatus",
+                  "insightType": "ApiCallRateInsight",
+                  "insightContext": {
+                      "statistics": {
+                          "baseline": {
+                              "average": 1.7561507937
+                          },
+                          "insight": {
+                              "average": 50
+                          },
+                          "insightDuration": 8
+                      }
+                  }
+              },
+              "eventCategory": "Insight"
+          }
+      ]
+  }
+result: ""
+results:
+- |
+  {
+      "eventVersion": "1.07",
+      "eventTime": "2019-10-17T10:05:00Z",
+      "awsRegion": "us-east-1",
+      "eventId": "aab985f2-3a56-48cc-a8a5-e0af77606f5f",
+      "eventType": "AwsCloudTrailInsight",
+      "recipientAccountId": "123456789012",
+      "sharedEventId": "12edc982-3348-4794-83d3-a3db26525049",
+      "insightDetails": {
+          "state": "Start",
+          "eventSource": "ssm.amazonaws.com",
+          "eventName": "UpdateInstanceAssociationStatus",
+          "insightType": "ApiCallRateInsight",
+          "insightContext": {
+              "statistics": {
+                  "baseline": {
+                      "average": 1.7561507937
+                  },
+                  "insight": {
+                      "average": 50.1
+                  }
+              }
+          }
+      },
+      "eventCategory": "Insight",
+      "p_event_time": "2019-10-17T10:05:00Z",
+      "p_any_aws_account_ids": ["123456789012"],
+      "p_log_type": "AWS.CloudTrailInsight"
+  }
+- |
+  {
+      "eventVersion": "1.07",
+      "eventTime": "2019-10-17T10:13:00Z",
+      "awsRegion": "us-east-1",
+      "eventId": "ce7b8ac1-3f89-4dae-8d2a-6560e32f591a",
+      "eventType": "AwsCloudTrailInsight",
+      "recipientAccountId": "123456789012",
+      "sharedEventId": "12edc982-3348-4794-83d3-a3db26525049",
+      "insightDetails": {
+          "state": "End",
+          "eventSource": "ssm.amazonaws.com",
+          "eventName": "UpdateInstanceAssociationStatus",
+          "insightType": "ApiCallRateInsight",
+          "insightContext": {
+              "statistics": {
+                  "baseline": {
+                      "average": 1.7561507937
+                  },
+                  "insight": {
+                      "average": 50
+                  },
+                  "insightDuration": 8
+              }
+          }
+      },
+      "eventCategory": "Insight",
+      "p_event_time": "2019-10-17T10:13:00Z",
+      "p_any_aws_account_ids": ["123456789012"],
+      "p_log_type": "AWS.CloudTrailInsight"
```

Comment:
@wey-chiang these are all tests copied over from the panther repo. These test the actual way that Panther parsers treat log inputs. Missing fields for example are here because the parser is not set up to catch this field as a domain.